scripts/develop.sh

```diff
@@ -97,14 +103,31 @@ CODER_DEV_SHIM="${PROJECT_ROOT}/scripts/coder-dev.sh"
 	fi
 
 	# Start the frontend once we have a template up and running
-	CODER_HOST=http://127.0.0.1:3000 yarn --cwd=./site dev || kill -INT -$$ &
+	CODER_HOST=http://127.0.0.1:3000 yarn --cwd=./site dev --host | {
+		while read -r line; do
+			echo "[SITE] $(date -Iseconds): $line"
+		done
+	} || kill -INT -$$ &
+
+	interfaces=(localhost)
+	if which ip >/dev/null 2>&1; then
+		# shellcheck disable=SC2207
+		interfaces+=($(ip a | awk '/inet / {print $2}' | cut -d/ -f1))
+	elif which ifconfig >/dev/null 2>&1; then
+		# shellcheck disable=SC2207
+		interfaces+=($(ifconfig | awk '/inet / {print $2}'))
+	fi
 
 	log
 	log "===================================================================="
 	log "==                                                                =="
 	log "==            Coder is now running in development mode.           =="
-	log "==                  API:    http://localhost:3000                 =="
-	log "==                  Web UI: http://localhost:8080                 =="
+	for iface in "${interfaces[@]}"; do
+		log "$(printf "==                  API:    http://%s:3000%$((26 - ${#iface}))s==" "$iface" "")"
```

Comment:
nit: please add a comment that `26 -` is to render spaces in TUI. You will save a few minutes of life of the next person :)